androidsvg/src/main/java/com/caverock/androidsvg/SVG.java

```diff
@@ -2417,16 +2457,21 @@ private SvgElementBase  getElementById(SvgContainer obj, String id)
    }
 
    //TODO Add remaining shapes, once their contains method has been implemented.
-   public List<SvgObject> getSVGObjectsByCoordinate(Point point) {
-      List<SvgObject> locations = new ArrayList<>();
+   public List<SVG.SvgObject> getSVGObjects() {

```

Comment:
It's good to leave TODOs, but until we do them, let's name our methods only according to what they currently do -- what do you think?